knative-operator/pkg/webhook/knativeeventing/webhook_validating_test.go

```diff
@@ -0,0 +1,93 @@
+package knativeeventing_test
```

Comment:
There is a lot of redundant code in here. I'm ok leaving it for now if it's too much effort to refactor, but I'd like it gone in a subsequent PR quickly.

Reply:
:+1: